client/app/competitor/page.tsx

```diff
@@ -12,6 +13,37 @@ import {
 } from '@/components/ui/accordion';
 import { ScrollArea } from '@/components/ui/scroll-area';
 import { Separator } from '@/components/ui/separator';
+import Leaderboard from '../leaderboard/page';
+
+const ObserveLeaderboardTab = () => {
+    const [isLeaderboard, setIsLeaderboard] = useState(false);
+
+    useEffect(() => {
+        const handleTabChange = (value: string) => {
+            if (value === 'Leaderboard') {
+                setIsLeaderboard(true);
+            } else {
+                setIsLeaderboard(false);
+            }
+        };
+
+        tabChangeEmitter.on('tabChange', handleTabChange);
+
+        return () => {
+            tabChangeEmitter.off('tabChange', handleTabChange);
+        };
+    }, []);
+
+    if (isLeaderboard) {
+        return (
+            <ScrollArea className="h-full w-full">
+                <Leaderboard />
```

Comment:
The timer is still shown in this view, but it may look nicer to hide it since we already have the timer in the bottom-left.